lms/djangoapps/course_api/blocks/views.py

```diff
@@ -240,6 +240,21 @@ def list(self, request, usage_key_string, hide_access_denials=False):  # pylint:
             # case we add the usual caching headers to the response.
             if params.cleaned_data.get('username', None) == '':
                 patch_response_headers(response)
+
+            if 'blocks' in response.data:
+                blocks = response.data['blocks']
+            else:
+                blocks = response.data
+
+            for block in blocks:
+                current_block = block
+                if isinstance(blocks, dict):
+                    current_block = blocks[block]
+                if 'children' in current_block:
+                    new_child_ids = [get_usage_key_hash(child) for child in current_block['children']]
+                    current_block['hash_children'] = new_child_ids
+                current_block['hash_key'] = get_usage_key_hash(current_block['id'])
+
```

Comment:
does the frontend code still need the course_blocks API response to be modified to contain the hash_key, or are the changes to the sequence metadata API sufficient?

Reply:
It needs it for when it is calls `normalizeBlocks` in `api.js`